comp/core/tagger/types/entity_id.go

```diff
@@ -0,0 +1,131 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package types defines types used by the Tagger component.
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	taggerutils "github.com/DataDog/datadog-agent/pkg/util/tagger"
+)
+
+const separator = separator
```

Comment:
Should it be `const separator = "://"` ?

Reply:
already fixed 👍